docs/api-reference/themes/antd/uiSchema.md

```diff
@@ -0,0 +1,22 @@
+# uiSchema Options for AntD
+
+You can customize the look of the form by passing options to Ant-Design theme fields.
+
+## formContext
+
+The formContext antd object accepts `descriptionLocation` property. It can be `'below' | 'tooltip'`, the default is `'below'`. Note that you should have antd 4.7+ to use `'tooltip'`.
+
+```
+descriptionLocation: Where to display the description, either 'below' or 'tooltip'
+```
+
+```jsx
+<Form
+  formContext={{
+    "antd": {
+      "descriptionLocation": "tooltip"
+    }
+    // other props...
+  }}
+/>
+```
```

Comment:
can you also move the `readonlyAsDisabled ` documentation from https://react-jsonschema-form.readthedocs.io/en/latest/api-reference/form-props/#formcontext to here? And replace that with "see [page X] for formContext customizations for the antd theme"